docs/pages/billing/invoices-and-receipts.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: View payment history, invoices, and receipts
+sidebar_title: Payment history, invoices, and receipts
+description: Learn how to view your account's payment history, download invoices and receipts, and request a refund for a charge.
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+**Receipts** in the Expo dashboard provide information about an account's payment history and access to invoices and receipts. It also provides information on payment dates, payment status, and the total amount for that payment. You can also request a refund for a charge if you believe it has been made in error.
+
+> **Note**: You can only access the **Receipts** if you have [Owner or Admin access](/accounts/working-together/#manage-access) to your account.
+
+## Receipts
+
+To view your account's payment history, click [**Receipts**](https://expo.dev/settings/receipts) in the navigation menu under **Account settings** or **Organization settings**.
+
+For example, an [Organization account's](/accounts/account-types/#organizations) receipts are shown below:
+
+<ImageSpotlight
+  alt="Receipts page in the Expo dashboard."
+  src="/static/images/billing/receipts-01.png"
+/>
+
+### Download and view an invoice
+
+To download and view an invoice for a billing period, go to **Receipts** and:
+
+- Click the **Date** for the billing period corresponding to the invoice. You will be navigated to a page hosted by Stripe. As an example, the March 22, 2024 invoice below links to this page:
+
+<ImageSpotlight
+  alt="Download invoice shown in the Stripe dashboard."
+  src="/static/images/billing/receipts-02.png"
+  style={{ maxWidth: 380 }}
+/>
+
+- Click **Download invoice**. You will receive a **.pdf** copy of the invoice.
```

Comment:
Nit: `you will receive a PDF copy`. We may also want to create an `<abbr>` element, although PDF is generally well-known.

Reply:
I'll update the instances of ".pdf" to PDF. I don't see the benefit of using <abbr> element here, as we generally use [common abbreviations as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations) without it.